src/controllers/connectionManager.ts

```diff
@@ -1514,7 +1514,11 @@ export default class ConnectionManager {
         }
     }
 
-    public async transferFileConnection(oldFileUri: string, newFileUri: string): Promise<void> {
+    public async transferFileConnection(
```

Comment:
Can you add a doc comment and maybe update the name to cloneConnectionForFile or duplicateFileConnection?